src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -283,45 +283,14 @@ internal static byte[] UrlDecode(byte[] bytes, int offset, int count)
                 }
                 else if (b == '%' && i < count - 2)
                 {
-                    if (bytes[pos + 1] == 'u' && i < count - 5)
-                    {
-                        int h1 = HttpEncoderUtility.HexToInt((char)bytes[pos + 2]);
```

Comment:
>If this code being removed didn't exist in desktop, it must have been explicitly added to core for some reason, right? Was it fixing a bug (I.e. the condition called out in this issue)?

@stephentoub  I have looked into NetFXDev1, but NCL doesn't own the code, and I can't find any information from our side